sources/api/host-containers/src/main.rs

```diff
@@ -434,22 +434,48 @@ where
     Ok(())
```

Comment:
There are some minor typos in the commit messages; I'd recommend the following to fix those and simplify a bit:
```
thar-be-settings: pass changed settings to restart commands

If a service is affected by changed settings, the changed settings are passed
to the restart commands of that service in an environment variable, separated
by spaces.  The variable is only set if there are changed settings that affect
the service, and only the settings relevant to that service are included.
```
and
```
host-containers: restart only modified host containers

If no settings have changed, say during startup, handle all host containers.
If settings are changed for particular host containers, handle only those
host containers.  If the settings are not host-container-specific, but affect
host containers overall, handle all host containers.
```